lib/routes/builds.js

```diff
@@ -381,8 +416,11 @@ module.exports = function (app) {
       await manager.untarPackage(contentData, untarPath);
```

Comment:
bleh. I don't relish doing this in the runtime service that is also returning build data.